src/v2/Apps/Fair/Routes/FairExhibitors.tsx

```diff
@@ -19,34 +14,22 @@ interface FairExhibitorsProps {
 const FairExhibitors: React.FC<FairExhibitorsProps> = ({ fair }) => {
   const { match } = useRouter()
   const { focused_exhibitor: focusedExhibitorID } = match.location.query
-  const offset = useExhibitorsTabOffset()
+  const { scrollTo } = useScrollTo({
+    selectorOrRef: `#jump--${focusedExhibitorID}`,
+    behavior: "smooth",
+    offset: 205,
```

Comment:
FIY an offset = 205 is used for "Exhibitors A-Z" in order to avoid such a problem (when clicking on a letter, it will scroll to the wrong position)


https://user-images.githubusercontent.com/3513494/146422227-b99fcc14-50e6-45b6-95ba-c5042e6454c8.mp4

Reply:
Though that offset is more a 'margin' since it should scroll exactly to the top of the desired element.

Reply:
If it's not then something is wrong.

Reply:
And if something is wrong we should fix the underlying problem here because if there's a bug and it gets fixed, this 'offset' is gonna be incorrect.

Reply:
> Though that offset is more a 'margin' since it should scroll exactly to the top of the desired element.

Yes, it will scroll to the desired element. We just have a fair header component (highlighted in red in the screenshot) and it overlaps the element we need (`offset 205` fixes this and for this reason we use such an offset)

<img width="762" alt="one x Artsy | Artsy 2021-12-16 20-59-57" src="https://user-images.githubusercontent.com/3513494/146424703-dd05222e-220e-44ca-bada-6a8dd798ba7e.png">

Reply:
Oh you're not using the StickyContext — that's why.

If you refactor this to use it you'll be fine.

Reply:
To be clear: once you wrap the page in a StickyContext you can also stack stickies. The useScrollTo hook is aware of the total offset of these.

Reply:
hmm, although I see that the Fair screen is already wrapped in `StickyProvider` [here](https://github.com/artsy/force/blob/3140ce4ed12c740231540511ff9b8e6188075dbe/src/v2/Apps/Fair/FairApp.tsx#L72). Or is it different?

Reply:
Ah, you're right — lemme check out the code and take a look.

Reply:
Discussed this moment in the DM